library/persistence/sqlite/src/cms_storage_sqlite_builder.e

```diff
@@ -65,10 +60,40 @@ feature -- Factory
 			create r.make ("authenticated")
 			r.add_permission ("create page")
 			r.add_permission ("edit page")
+			r.add_permission ("delete page")
 			a_storage.save_user_role (r)
 
-				-- Test custom value
```

Comment:
I know I left two line of tests for set_custom_value, but this was an error from me, those classes should not contain any "test" purpose data/code.

We should really move them to autotest classes.

Here, we could keep  users + roles initialization .. as they are required.

we could also move that to the USER_MODULE implementation.